Tests/TensorFlowTests/TensorAutoDiffTests.swift

```diff
@@ -16,7 +16,7 @@ import XCTest
 
 @testable import TensorFlow
 
-let cube: @differentiable (Tensor<Float>) -> Tensor<Float> = { ($0 * $0 * $0) }
+let cube: @differentiable(Tensor<Float>) -> Tensor<Float> = { ($0 * $0 * $0) }
```

Comment:
Hmm, did `swift-format` make this change? Maybe we can ask `swift-format` to fix this formatting (don't delete space after `@differentiable` in function types).
```suggestion
let cube: @differentiable (Tensor<Float>) -> Tensor<Float> = { ($0 * $0 * $0) }
```